packages/pwa-kit-build/src/configs/webpack/config.js

```diff
@@ -221,7 +226,11 @@ const client =
                 entry: {
                     main: './app/main'
                 },
-                plugins: [...config.plugins, new LoadablePlugin({writeToDisk: true})],
+                plugins: [
+                    ...config.plugins,
+                    new LoadablePlugin({writeToDisk: true}),
+                    analyzeBundle && getBundleAnalyzerPlugin('main')
```

Comment:
How about we make the names consistent? I see there are:
- `const client` - variable name
- `name: 'client'` - config name
- `getBundleAnalyzerPlugin('main')`- bundle report name

IMO it's better if we use the name "client" in this case ← following the config name

Reply:
I've renamed to use `client`